desk/app/groups.hoon

```diff
@@ -1107,47 +1136,59 @@
       [%x %v2 %init ~]
     =/  groups-light-ui-7=(map flag:v7:gv group-ui:v7:gv)
       %-  ~(urn by groups)
-      |=  [=flag:g =net:v7:gv =group:v7:gv]
-      =*  light-group  (drop-seats:group:v7:gc group our.bowl)
-      =+  (group-ui:group:v7:gc net light-group)
+      |=  [=flag:g =net:v9:gv =group:v9:gv]
+      =*  light-group  (drop-seats:group:v9:gc group our.bowl)
+      =+  (group-ui:v7:group:v9:gc net light-group)
       ::  restore member count after dropping seats
       ::TODO is this correct in other scries?
       ::
       -(member-count ~(wyt by seats.group))
     ``noun+!>([groups-light-ui-7 `foreigns:v8:gv`foreigns])
+  ::
+      [%x %v3 %init ~]
+    =/  groups-light-ui-9=(map flag:v9:gv group-ui:v9:gv)
+      %-  ~(urn by groups)
+      |=  [=flag:g =net:v9:gv =group:v9:gv]
+      =*  light-group  (drop-seats:group:v9:gc group our.bowl)
+      =+  (group-ui:group:v9:gc net light-group)
+      ::  restore member count after dropping seats
+      ::TODO is this correct in other scries?
```

Comment:
We.. copy this comment over instead of checking it?

Not obvious to me what's meant by "other scries" here, but certainly at this point it'd be part of the known/accepted behavior, and so will only change by version going forward?